pkg/kube/kube.go

```diff
@@ -207,7 +207,7 @@ func (k *KubeClient) runKubectlCmdWithInput(args []string, input []byte) ([]byte
 	return out, nil
 }
 func (k *KubeClient) LabelExistsInCluster(ctx context.Context, label string) error {
-	cmd := []string{"get", "app", "-l", fmt.Sprintf("weave-gitops.weave.works/app-identifier=%s", label)}
+	cmd := []string{"get", "app", "-l", fmt.Sprintf("weave-gitops.weave.works/app-identifier=%s", label), "--all-namespaces"}
```

Comment:
I wonder if it would be ok to make the name of the function and the label parameter a bit more descriptive 🙂 . It looks like we are looking for a very specific label value rather than a label itself.

Reply:
I think if we end using more labels we will for sure need to do that. I am not sure if we ever plan on doing that though